draft-irtf-cfrg-ristretto.xml

```diff
@@ -27,8 +27,7 @@
 <t>This memo specifies a prime-order group, ristretto255, suitable for
 safely implementing higher-level and complex cryptographic protocols.
 The ristretto255 group can be implemented using Curve25519, allowing
-existing Curve25519 implementations to be reused and extended to provide
-a prime-order group.</t>
+existing Curve25519 implementation reuse for a prime-order group.</t>
```

Comment:
Hmm, I think that "reused and extended" is better language, because it makes it clear that Ristretto does not come automatically (you cannot reuse a Curve25519 implementation directly, but you must extend it very slightly using the Ristretto functions).

Reply:
Good point! I was mainly aiming to reduce text, so, feel free to ignore. :)